usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -87,17 +88,37 @@
                 0
             ) AS NUMERIC(23, 2)
         ) AS face_value_loan_guarantee,
-        COUNT(*) AS counts
+        COUNT(*) AS counts,
+        CAST(
+            COALESCE(
+                SUM(matching_awards.total_outlays),
+                0
+            ) AS NUMERIC(23, 2)
+        ) AS total_outlays
     FROM
         int.transaction_normalized
+    JOIN
```

Comment:
Is this join being used? Looks like the reference to `as2` below is a completely separate instance of Award Search

Reply:
It wasn't being used. I have taken that line out.